packages/enclave/src/controllers/eth/wallets.controller.ts

```diff
@@ -118,6 +118,11 @@ export default class WalletsController
       this.promiseWrapper(this.replaceMasterWalletTransaction, 201)
     );
 
+    this.router.post(
+      `${this.path}/:masterWalletId/transactions/resend`,
+      this.promiseWrapper(this.resendMasterWalletTransaction, 201)
```

Comment:
In wallet API, resending transaction API returns status code `202`. So, I think we need to return status code `202`, too.

Reply:
fixed it